src/issues/issueFeatureRegistrar.ts

```diff
@@ -1150,19 +1229,17 @@ ${body ?? ''}\n
 		return linkUri.with({ authority, path: linkPath }).toString();
 	}
 
-	async copyPermalink(repositoriesManager: RepositoriesManager, fileUri?: vscode.Uri) {
-		const link = await this.getPermalinkWithError(repositoriesManager, fileUri);
+	async copyPermalink(repositoriesManager: RepositoriesManager, context?: LinkContext, includeRange = true, contextualizeLink = false, includeFile = true) {
+		const link = await this.getPermalinkWithError(repositoriesManager, context, includeRange, includeFile);
 		if (link.permalink) {
-			return vscode.env.clipboard.writeText(
-				link.originalFile ? (await this.getContextualizedLink(link.originalFile, link.permalink)) : link.permalink);
+			return vscode.env.clipboard.writeText(contextualizeLink && link.originalFile ? await this.getContextualizedLink(link.originalFile, link.permalink) : link.permalink);
 		}
 	}
 
-	async copyHeadLink(fileUri?: vscode.Uri) {
-		const link = await this.getHeadLinkWithError(fileUri);
+	async copyHeadLink(fileUri?: vscode.Uri, includeRange = true) {
+		const link = await this.getHeadLinkWithError(fileUri, includeRange);
 		if (link.permalink) {
-			return vscode.env.clipboard.writeText(
-				link.originalFile ? (await this.getContextualizedLink(link.originalFile, link.permalink)) : link.permalink);
+			return vscode.env.clipboard.writeText(link.permalink);
```

Comment:
Why not contextualize this one now?

Reply:
Given that we now have vscode.dev commands registered even when in vscode.dev (and github.dev commands registered in github.dev), I think it would be redundant to also offer contextualized GitHub permalinks, so I'm proposing to convert the GitHub permalink actions to always return github.com links. The only scenario where these links are still contextualized is for the github.dev links that GHPRI will register with this PR.